packages/terra-framework-docs/src/terra-dev-site/doc/worklist-data-grid/Examples.2/WorklistDataGridWithRowSelection.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import WorklistDataGrid from 'terra-worklist-data-grid';
+import gridDataJSON from './gridData.json';
```

Comment:
When I select rows in the 'With Selection' example and then switch to the default example, the rows are still selected. I assume it's importing this json as the same object in both examples which is then being mutated by this example.

At any rate, since the important part of the example is contained in this object, it should be included in this file so consumers can see it on the doc site. The example data should also be replaced with Worklist-style data rather than Flowsheet style and pared down to only a few rows so as not to make the example needlessly verbose. Defining the data inline with the example will solve the issue above.

It's not a great practice to be mutating and and pulling state from an external object instead of using React state (as illustrated it causes problems). Since this is for public documentation, I think it should stick to best practices.